bin/desi_dailyproc

```diff
@@ -83,16 +92,30 @@ def parse(options=None):
         args = parser.parse_args()
     else:
         args = parser.parse_args(options)
+
+    if args.catchupfile.lower() != 'none':
+        args.catchup = True
+
+    if args.cameras.lower() != 'none':
```

Comment:
See comment below about default `None` vs. `"None"`, which applies to both `args.catchupfile` and `args.cameras`.